lesion_segmentation_protocol.md

```diff
@@ -0,0 +1,31 @@
+# Lesion segmentation protocol:
+
+The following details the protocol for Multiple Sclerosis (MS) lesion segmentation in the spinal cord.
+Imaging the spinal cord is often essential to confirm the diagnosis of MS. That is because the lesions of the spinal cord are included in the McDonald diagnosis criteria, which studies dissemination in space and in time [(Thompson et al. 2018)](https://pubmed.ncbi.nlm.nih.gov/29275977/). While the MAGNIMS-CMSC-NAIMS working group recommends to use at least two sagittal images for MS diagnosis, still, axial imaging is mentioned as optional in international imaging guidelines [(Wattjes et al. 2021)](https://pubmed.ncbi.nlm.nih.gov/34139157/).
+For detecting MS lesions in the spinal cord, two main contrasts emerge: PSIR and STIR contrasts. New studies [(Peters et al. 2024)](https://pubmed.ncbi.nlm.nih.gov/38289376/)[(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that using PSIR contrasts improved MS lesion detection in the spinal cord. [(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that the PSIR contrast showed a higher signal-to-noise (SNR) ratio compared to the STIR contrast. 
```

Comment:
Maybe my suggestion is not up-to-date, but I have doubts/questions abouts STIR sequences being frequently used for assessing MS lesions, at least in standard/basics protocols (even if mentioned in the picture showing example of of clinically used sequences for MS). From my understanding, the most useful sequence for detecting MS lesions in clinical practice is proton density (PD), but probably that changes from sites/radiologists perspective.
<img width="339" alt="basic_sequences" src="https://github.com/ivadomed/ms-lesion-agnostic/assets/142258732/afe5cd16-69bd-43e1-b4a2-a3e9b7c4a3ff">
According to the articles, PSIR sequences seem great! However, here are some potential precisions I would address :
1-Peters and al : PSIR compared to STIR (not the best...) and T2;
2-Fechner : PSIR compared to T2 and T1C+ sequences (not compared to PD sequences...)
```suggestion
For detecting MS lesions in the spinal cord, two main contrasts emerge: PSIR and STIR contrasts. New studies [(Peters et al. 2024)](https://pubmed.ncbi.nlm.nih.gov/38289376/)[(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that using PSIR contrasts improved MS lesion detection in the spinal cord. [(Fechner et al. 2019)](https://pubmed.ncbi.nlm.nih.gov/30679225/) showed that the PSIR contrast showed a higher signal-to-noise (SNR) ratio compared to the STIR contrast. 
```